crates/anvil/src/cmd.rs

```diff
@@ -599,6 +600,10 @@ pub struct AnvilEvmArgs {
     /// Enable Odyssey features
     #[arg(long, alias = "alphanet")]
     pub odyssey: bool,
+
+    /// Run a Celo chain
+    #[arg(long)]
+    pub celo: bool,
```

Comment:
hmm, not convinced we want to add a new arg for every network that we can add custom precompiles for.

should we perhaps derive this from the chainid, although an additional celo flag doesnt really hurt

Reply:
wouldn't be better to have a `--precompiles` arg and enable the precompiles based on the passed value? like `--precompiles celo` or `--precompiles arbitrum`